django_rq/workers.py

```diff
@@ -38,6 +39,11 @@ def get_worker_class(worker_class=None):
     return worker_class
 
 
+def activate_localization():
+    if getattr(settings, 'RQ_USE_LOCALIZATION', False):
```

Comment:
Could you please move this variable to DjangoRQ's `settings.py`?

Reply:
Done